src/mod/transfer.mod/language/transfer.portuguese.lang

```diff
@@ -11,10 +11,10 @@
 0xf06,NOTICE %s :Erro de DCC socket; abortando ficheiros em espera.\n
 0xf07,Abortada obtenção dcc %s: Ficheiro vazio!
 0xf08,NOTICE %s :Ficheiro %s está vazio, abortando transferência.\n
-0xf09,  Send to  %s  Ficheiro\n
+0xf09,  Enviando para  %s  Ficheiro\n
 0xf0a,  ---------%s  --------------------\n
```

Comment:
You'll want to add more "-" so the %s match up again with 0xf09.
And add more spaces at 0xf0b and 0xf0c so the third %s matches up with the %s at 0xf09.